runtime/queries/c/injections.scm

```diff
@@ -4,6 +4,28 @@
 ((comment) @injection.content
  (#set! injection.language "comment"))
 
+((call_expression
+  function: (identifier) @_function
+  arguments: (argument_list
+               . (string_literal (string_content) @injection.content)))
+ (#any-of? @_function "printf" "scanf" "printf_s" "scanf_s")
+ (#set! injection.language "printf"))
+
+((call_expression
+  function: (identifier) @_function
+  arguments: (argument_list
+               (_) . (string_literal (string_content) @injection.content)))
+ (#any-of? @_function "fprintf" "fscanf" "fprintf_s" "fscanf_s"
+                      "sprintf" "sscanf" "sscanf_s")
+ (#set! injection.language "printf"))
+
+((call_expression
+  function: (identifier) @_function
+  arguments: (argument_list
+               (_) . (_) . (string_literal (string_content) @injection.content)))
+ (#any-of? @_function "snprintf" "sprintf_s" "snprintf_s")
+ (#set! injection.language "printf"))
```

Comment:
there are a lot more than these

`man 3 printf` 

Missing:
dprintf (2nd arg)
vprintf (1st)
vfprintf (2nd)
vdprintf (2nd)
vsprintf (2nd)
vsnprintf (3rd)

Reply:
Just gonna pretend MS CRT is not a thing.